### PR TITLE
docs(top-app-bar): Add metadata for catalog site

### PR DESCRIPTION
**Pull request by kfranqueiro**
We held off adding this until several features of Top App Bar were fleshed out. Now that we've added the deprecation notice to Toolbar, it makes sense to make this visible in the catalog.

### CODE REVIEW

File: packages/mdc-top-app-bar/README.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+excerpt: "A container for items such as application title, navigation icon, and action items."
+iconId: toolbar
+path: /catalog/top-app-bar/
+-->
+
 # Top App Bar
 
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/top-app-bar/index.html">
```
Comment (williamernest) [inline packages/mdc-top-app-bar/README.md]: We don't have a `top-app-bar` folder yet. We probably will in the future, but for now it's just `/top-app-bar.html`

File: packages/mdc-top-app-bar/README.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/top-app-bar.png" width="494" alt="Top App Bar screenshot">
```
Comment (williamernest) [inline packages/mdc-top-app-bar/README.md]: Does thing `png` exist already?

Comment (kfranqueiro) [reply packages/mdc-top-app-bar/README.md]: I simultaneously have a PR open in the site generator repo to add it.

File: packages/mdc-top-app-bar/README.md (hunk 1 above)
Comment (kfranqueiro) [reply packages/mdc-top-app-bar/README.md]: Oops. :facepalm:

**Pull request merged**